app/mailers/club_application_mailer.rb

```diff
@@ -12,9 +12,9 @@ def applicant_confirmation(application)
   def admin_notification(application)
     @application = application
 
-    to = Mail::Address.new 'team@hackedu.us'
+    to = Mail::Address.new 'team@hackclub.io'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.